resources/views/users/edit.blade.php

```diff
@@ -12,7 +12,7 @@
                     <p>Editing: {{$user->name}}</p>
                     <p>Email: {{$user->email}}</p>
                     <p>Badge Number: {{$user->badge_number}}</p>
-                    <p>Team: {{$user->team->name}}</p>
```

Comment:
@drraccoony I'll need to submit a commit to uncomment this out, missed this during the rebase/merge. But I want to make sure there aren't any other changes which I need to make first.